molgenis-security-core/src/main/java/org/molgenis/security/core/utils/SecurityUtils.java

```diff
@@ -67,13 +67,14 @@ public static boolean currentUserHasRole(String... roles)
 		if (authentication != null)
 		{
 			Collection<? extends GrantedAuthority> authorities = authentication.getAuthorities();
+			if (authorities == null) throw new IllegalStateException("No current user logged in");
```

Comment:
Bit of a strange message, maybe something like: "No user currently logged in" or "No currently logged in user"